build.gradle

```diff
@@ -76,15 +74,14 @@ dependencies {
     api 'com.fasterxml.jackson.core:jackson-databind:2.11.0'
 
     // Preconditions
-    api 'com.google.guava:guava:29.0-jre'
+    api 'com.google.guava:guava:31.0-jre'
 
     // Testing libraries
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
-    testCompileOnly 'junit:junit:4.13'
-    testRuntimeOnly 'org.junit.vintage:junit-vintage-engine'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
 
     testImplementation 'org.hamcrest:hamcrest:2.2'
-    testImplementation 'com.github.tomakehurst:wiremock:2.26.3'
+    testImplementation 'com.github.tomakehurst:wiremock-jre8:2.31.0'
     testImplementation 'org.mockito:mockito-core:3.3.3'
     testImplementation 'com.squareup.okhttp3:mockwebserver:4.7.2'
```

Comment:
Definitely too many mock dependencies. A small interface and an integration test with Micro would free us with these dependencies to bump every few months.

Maybe we can bump these two too?!